Fixed typo (#4889)
segmentio/segment-docs: src/connections/spec/best-practices-identify.md

- When first you create a user (and so it is assigned a `userId`)
- When a user changes information in their profile
- When a user logs in

> - Optionally, when you call `identify` upon loading any pages that aren't accessible by a logged in user

Reviewer: ```suggestion
- *(Optional)* Upon loading any pages that are accessible by a logged in user
```

Reviewer: @jfoskin I believe "are" is correct
